test/test_compressed_support.py

```diff
@@ -0,0 +1,97 @@
+#-------------------------------------------------------------------------------
+# elftools tests
+#
+# Pierre-Marie de Rodat (pmderodat@kawie.fr)
+# This code is in the public domain
```

Comment:
Also, does readelf support this? I wonder if it's possible to have a readelf-comparison test for a file with compressed sections